util/packager.js

```diff
@@ -1,82 +1,129 @@
+// At the same time, save the unencoded ELM JSON off into .js files so it is
 // easier for the dashboard to import them.
-const inputPath = 'cql';
-const fshPath = join('fsh-tank','input','fsh');
-const fhirPath = join('fsh-tank','fsh-generated','resources');
-let exportBase = join('dist','cql');
-if (!existsSync(exportBase)) mkdirSync(exportBase, {recursive: true});
+const inputPath = "cql";
+const fshPath = join("fsh-tank", "input", "fsh");
+const fhirPath = join("fsh-tank", "fsh-generated", "resources");
+const bundlePath = join("util", "resources");
+
+let exportBase = join("dist", "cql");
+if (!existsSync(exportBase)) mkdirSync(exportBase, { recursive: true });
 let files = readdirSync(inputPath);
+
 for (const file of files) {
   if (/(\.json)$/.test(file)) {
     const contentString = readFileSync(inputPath + sep + file);
+    // read the cql data for the elm
+    const cql = readFileSync(inputPath + sep + file.replace(".json", ".cql"));
     // Create a .js version of the ELM .json file
     try {
-      const varName = file.slice(0,-5).replace(/-/g,'v').replace(/\./g,'');
-      const updatedContents = 'export const ' + 
-        varName + 
-        ' = ' +
-        contentString;
-      const jsFile = file.replace('.json','.mjs');
-      writeFileSync(join(exportBase,jsFile), updatedContents);
-    } catch(e) {
+      const varName = file.slice(0, -5).replace(/-/g, "v").replace(/\./g, "");
+      const updatedContents = "export const " + varName + " = " + contentString;
+      const jsFile = file.replace(".json", ".mjs");
+      writeFileSync(join(exportBase, jsFile), updatedContents);
+    } catch (e) {
+      console.log(e);
+
       continue;
     }
     // Replace the base64 string in the corresponding library .fsh file
-    const contentBase64 = Buffer.from(contentString).toString('base64');
-    const fshFile = file.replace('.json','.fsh');
+    const contentBase64 = Buffer.from(contentString).toString("base64");
+    const fshFile = file.replace(".json", ".fsh");
     try {
-      const outputContents = readFileSync(fshPath + sep + fshFile).toString();
-      const updatedContents = outputContents.replace(
-        /\* content\[\=\]\.data \= \".+\"/,
-        '* content[=].data = \"' + contentBase64 + '\"'
+      let outputContents = readFileSync(fshPath + sep + fshFile).toString();
+      let lines = outputContents.split("\n");
+
+      // Find the index of the line starting with '* description'
+      const descriptionIndex = lines.findIndex((line) =>
+        line.trim().startsWith("* description")
       );
+      if (descriptionIndex !== -1) {
+        lines = lines.slice(0, descriptionIndex + 1);
+      }
+
+      const updatedContents = `${lines.join('\n')}
+* content[cqlContent].contentType = urn:ietf:bcp:13#text/cql "text/cql"
+* content[cqlContent].data = "${cql.toString("base64")}"
+* content[1].contentType = urn:ietf:bcp:13#application/elm+json "application/elm+json"
+* content[1].data = "${contentBase64}"
+
+`;
```

Comment:
If I'm understanding correctly, this adds a dependency for the content field to be immediately after the description field, which may not always be the case. Should we restore the old behavior where we explicitly replace the content.data elements?

Reply:
it's either after the description field or tacked on to the end.  I'm not sure why this change was made, you'd have to ask Rob.  Though it shouldn't matter, the FSH files are local.  If a file is missing the description or has the description in the wrong place we can just fix it.